test/test_aggregate_graphs.py

```diff
@@ -127,7 +127,6 @@ class GraphAggregates2(unittest.TestCase):
     def setUp(self):
         memStore = plugin.get('SQLAlchemy', Store)(
             identifier="rdflib_test", configuration=Literal("sqlite://"))
-        print("ASDF")
```

Comment:
left those dangling prints in my previous PR, caught them here with help of flake8-print that now runs as part of CI!